python/django_bolt/api.py

```
@@ -1772,15 +1791,19 @@ async def execute_multi(handler: Callable, request: dict[str, Any]) -> ResponseW
                 if is_async:
                     return await _dispatch_multi_async(await handler(*args, **kwargs))
                 if is_blocking:
-                    result = await sync_to_thread(handler, *args, **kwargs)
-                    return await sync_to_thread(_dispatch_multi_sync, result)
+                    # ONE thread hop for handler + serialization (two hops =
+                    # double executor wait + double pool contention).
+                    def _run_blocking_multi():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_run_blocking_multi`

(ANN202)